--- slapd.ldapv3.conf.tmpl.orig	2006-11-19 22:39:38.000000000 +0100
+++ slapd.ldapv3.conf.tmpl	2006-11-19 22:41:48.000000000 +0100
@@ -7,10 +7,10 @@
 include         /etc/ldap/schema/nis.schema
 include         /etc/ldap/schema/misc.schema
 include         /etc/ldap/schema/inetorgperson.schema
-include         /etc/ldap/schema/dnszone.schema
-include         /etc/ldap/schema/pureftpd.schema
-include         /etc/ldap/schema/ispman.schema
-include         /etc/ldap/schema/RADIUS-LDAPv3.schema
+include         /usr/share/ispman/conf/schema/dnszone.schema
+include         /usr/share/ispman/conf/schema/pureftpd.schema
+include         /usr/share/ispman/conf/schema/ispman.schema
+include         /usr/share/ispman/conf/schema/RADIUS-LDAPv3.schema
 
 
 # Define global ACLs to disable default read access.
@@ -19,8 +19,8 @@
 # service AND an understanding of referrals.
 #referral	ldap://root.openldap.org
 
-pidfile		/var/run/slapd.pid
-argsfile	/var/run/slapd.args
+pidfile		/var/run/slapd/slapd.pid
+argsfile	/var/run/slapd/slapd.args
 
 # Load dynamic backend modules:
 modulepath	/usr/lib/ldap
