--- /etc/nsswitch.conf.orig     2006-11-28 03:18:35.000000000 +0100
+++ /etc/nsswitch.conf  2006-11-29 00:36:16.379084352 +0100
@@ -4,9 +4,9 @@
 # If you have the `glibc-doc' and `info' packages installed, try:
 # `info libc "Name Service Switch"' for information about this file.

-passwd:         compat
-group:          compat
-shadow:         compat
+passwd:         files ldap
+group:          files ldap
+shadow:         files ldap

 hosts:          files dns
 networks:       files

